go-controller/pkg/ovn/base_network_controller_kubevirt.go

```diff
@@ -0,0 +1,52 @@
+package ovn
+
+import (
+	"fmt"
+
+	"github.com/ovn-org/ovn-kubernetes/go-controller/pkg/kubevirt"
+	"github.com/ovn-org/ovn-kubernetes/go-controller/pkg/util"
+	corev1 "k8s.io/api/core/v1"
+	"k8s.io/client-go/util/retry"
+)
+
+// getSwitchNames at some kubevirt scenarios the switch owner the IP is
+// different from the one running the pod
+func (bnc *BaseNetworkController) getSwitchNames(pod *corev1.Pod) (string, string, error) {
```

Comment:
nit: maybe it's just me but this signature's return type is confusing; could you add a struct that groups the "original" switch name with the "current" switch name ? 

Maybe add it a utility func to know if said pod has migrated to a different node ? 

At least indicate in the docstring which of the strings is the current and original switch name.

Reply:
Doing a struct with docstring and a stringer function.